api/envoy/service/auth/v2/external_auth.proto

```diff
@@ -61,7 +61,8 @@ message OkHttpResponse {
 
 // Intended for gRPC and Network Authorization servers `only`.
 message CheckResponse {
-  // Status `OK` allows the request. Any other status indicates the request should be denied.
+  // Status `OK` allows the request. Status `UNKNOWN` causes Envoy to abort. Any other status
```

Comment:
what does abort mean here?

Reply:
"abort" means "exit with a stack trace", which I guess is probably a bug (https://github.com/envoyproxy/envoy/issues/6210).